main_test.go

```diff
@@ -0,0 +1,171 @@
+// Copyright 2021 Google LLC
```

Comment:
Moved this in the root folder since we can't call main() form any other package in go.